cmd/derpprobe/derpprobe.go

```diff
@@ -50,6 +52,7 @@ var (
 	lastDERPMap   *tailcfg.DERPMap
 	lastDERPMapAt time.Time
 	certs         = map[string]*x509.Certificate{}
+	reportedBad   bool
```

Comment:
this makes it sound like a one-time false=>true variable. Maybe "inBadState"? Or at least a comment like `// whether we're currently in report-worthy failure state`

Reply:
Done (also rearranged to not be a global now)